### PR TITLE
Remove "Superusers" from organization detail page

### PR DESCRIPTION
**Pull request by midmarch**
Changes:

- Rename label "Administrating positions" to "Assigned Superusers"
- Remove Superuser listing of organisation positions

Closes [AB#793](https://dev.azure.com/ncia-anet/2aa083a5-af3d-44e1-8c7b-6e9e6b124d91/_workitems/edit/793)

#### System admin changes
- [x] anet.yml or anet-dictionary.yml needs change
- [ ] db needs migration
- [x] documentation has changed
- [ ] graphql schema has changed

### Checklist
  - [x] Described the user behavior in PR body
  - [x] Referenced/updated all related issues
  - [x] commits follow a `repo#issue: Title` title format and [these 7 rules](https://chris.beams.io/posts/git-commit/)
  - [x] commits have a [clean history](https://epage.github.io/dev/commits/), otherwise PR may be squash-merged
  - [x] Added and/or updated unit tests
  - [x] Added and/or updated e2e tests
  - [ ] Added and/or updated data migrations
  - [x] Updated documentation
  - [x] Resolved all build errors and warnings
  - [ ] Opened debt issues for anything not resolved here

### CODE REVIEW

File: client/src/components/previews/OrganizationPreview.js [hunk 1]
```diff
@@ -109,14 +109,6 @@ const OrganizationPreview = ({ className, uuid }) => {
     ? Settings.fields.principal.org
     : Settings.fields.advisor.org
 
-  const superUsers = organization.positions.filter(
```
Comment (gjvoosten) [inline client/src/components/previews/OrganizationPreview.js:112]: You _probably_ want to make the same changes to pages/organizations/Show.js…

**Review by gjvoosten — CHANGES_REQUESTED** [review]

File: anet-dictionary.yml [hunk 2]
```diff
@@ -1199,7 +1199,7 @@ fields:
       location:
         filter: [ADVISOR_LOCATION]
       administratingPositions:
-        label: Administrating positions
+        label: Assigned Super Users
```
Comment (gjvoosten) [inline anet-dictionary.yml:1202]: Please keep `testDictionaries/no-custom-fields.yml` in line with this change.

File: anet-dictionary.yml [hunk 3]
```diff
@@ -1428,7 +1428,7 @@ fields:
       location:
         filter: [PRINCIPAL_LOCATION]
       administratingPositions:
-        label: Administrating positions
+        label: Assigned Super Users
```
Comment (gjvoosten) [inline anet-dictionary.yml:1431]: Please keep `testDictionaries/no-custom-fields.yml` in line with this change.

File: client/tests/e2e/permissions.js [hunk 4]
```diff
@@ -54,7 +54,7 @@ test.serial("checking super user permissions", async t => {
 
   await validateUserCanEditUserForCurrentPage(t)
 
-  // User is super user, they may edit position of type super user for
+  // User is super-user, they may edit position of type super user for
```
Comment (gjvoosten) [inline client/tests/e2e/permissions.js:57]: If you change "super user" to "super-user", at least be consistent on the rest of this line…

File: client/src/pages/organizations/Laydown.js [hunk 5]
```diff
@@ -131,7 +131,7 @@ const OrganizationLaydown = ({ organization, refetch }) => {
           )
         }
       >
-        <PositionTable positions={organization.administratingPositions} />
+        <PositionTable positions={organization.administratingPositions} id="superUserTable"/>
```
Comment (gjvoosten) [inline client/src/pages/organizations/Laydown.js:134]: Code is not properly formatted.

File: client/tests/e2e/permissions.js [hunk 6]
```diff
@@ -123,12 +123,9 @@ test.serial("checking super user permissions", async t => {
...
 
-  await assertElementNotPresent(
-    t,
-    ".edit-position",
-    "super user should not be able to edit positions of the organization their position is not administrating",
-    shortWaitMs
-  )
+  const $editPositionButton = await $(".edit-position")
+  await t.context.driver.wait(t.context.until.elementIsVisible($editPositionButton))
+  t.pass("Jacob should be able to edit his own organization (\"EF 2.2\")")
```
Comment (gjvoosten) [inline client/tests/e2e/permissions.js:128]: Lines are not properly formatted.

File: client/tests/e2e/permissions.js [hunk 7]
```diff
@@ -234,12 +231,12 @@ test.serial("checking admin permissions", async t => {
 
   await t.context.pageHelpers.clickMenuLinksButton()
   await t.context.pageHelpers.clickMyOrgLink()
-  const $arthurLink = await findSuperUserLink(t, "CIV DMIN, Arthur")
-  await $arthurLink.click()
-  await t.context.driver.wait(t.context.until.stalenessOf($arthurLink))
+
+  const element = await t.context.driver.findElement(By.linkText("CIV DMIN, Arthur"))
```
Comment (gjvoosten) [inline client/tests/e2e/permissions.js:235]: Line is not properly formatted.

**Review by gjvoosten — CHANGES_REQUESTED** [review]

File: client/tests/e2e/permissions.js [hunk 8]
```diff
@@ -332,7 +329,7 @@ function validateUserCannotEditOtherUser(
 }
 
 async function findSuperUserLink(t, desiredSuperUserName) {
-  const $superUserLinks = await t.context.$$("[name=superUsers] p a")
+  const $superUserLinks = await t.context.$$("[id=superUserTable] tbody tr td:nth-child(4) span a")
```
[the remark this answers is not in the document]
Comment (midmarch) [reply client/tests/e2e/permissions.js]: As the existing indentention appeared a bit inconsistent I tried to allign those as well.

File: client/tests/e2e/permissions.js [hunk 9]
```diff
@@ -28,8 +28,8 @@ test.serial("checking super user permissions", async t => {
   const $tooltip = await $("#role_ADVISOR_tooltip")
   t.regex(
     await $tooltip.getAttribute("title"),
-    /^Super users cannot create .*$/,
-    "Expected tooltip for super users"
+    /^Super-users cannot create .*$/,
```
Comment (gjvoosten) [inline client/tests/e2e/permissions.js:31]: You should update the code as well if you change this (see `client/src/pages/people/Form.js`).

**Review by gjvoosten — CHANGES_REQUESTED** [review]

**Review by gjvoosten — APPROVED** [review]

**Pull request merged**